zipline/utils/cli.py

```diff
@@ -188,9 +225,13 @@ def run_pipeline(print_algo=True, **kwargs):
     algo = zipline.TradingAlgorithm(script=algo_text,
                                     namespace=kwargs.get('namespace', {}),
                                     capital_base=float(kwargs['capital_base']),
-                                    algo_filename=kwargs.get('algofile'))
+                                    algo_filename=kwargs.get('algofile'),
+                                    asset_metadata=asset_metadata,
+                                    identifiers=symbols,
+                                    start=start,
+                                    end=end)
 
-    perf = algo.run(source)
+    perf = algo.run(source, overwrite_sim_params=False)
```

Comment:
Sorry, this was a carry-over from testing. We do now accept start and end dates properly from the CLI, though, so I've modified it to only overwrite if no start and end dates are provided.